### PR TITLE
Jenkins Plugin with region

### PR DESCRIPTION
**Pull request by yashviagrawal**

### CODE REVIEW

File: src/main/resources/index.jelly [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <?jelly escape-by-default='true'?>
 <div>
-    The Appknox Security Scan Plugin allows you to perform Appknox security scan on your mobile application binary.
+    It allows you to perform Appknox security scan on your mobile application binary.
```
Comment (ginilpg) [inline src/main/resources/index.jelly:3]: Can you change this line as "This plugin allows you .. "

File: src/main/resources/io/jenkins/plugins/scanner/AppknoxScanner/help-apiHost.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div>
+    Select the Appknox Regions. Select this option if you're using a different region
```
Comment (ginilpg) [inline src/main/resources/io/jenkins/plugins/scanner/AppknoxScanner/help-apiHost.html:2]: @yashviagrawal Change this as " Select a region if you're using a different cloud instance of Appknox than the default global region."

**Review by ginilpg — CHANGES_REQUESTED** [review]

File: src/main/resources/io/jenkins/plugins/scanner/AppknoxScanner/help-apiHost.html (hunk 2 above)
Comment (yashviagrawal) [reply src/main/resources/io/jenkins/plugins/scanner/AppknoxScanner/help-apiHost.html:2]: Done

File: src/main/resources/index.jelly (hunk 1 above)
Comment (yashviagrawal) [reply src/main/resources/index.jelly:3]: Done

**Review by ginilpg — APPROVED** [review]

**Pull request merged**